packaging/nnstreamer.spec

```diff
@@ -85,7 +85,10 @@ BuildRequires:	pkgconfig(edgetpu)
 %endif
 
 %if 0%{?testcoverage}
+# the latest version of lcov requires extra perl modules
```

Comment:
Really? ```platform/upstream/lcov``` has no changes for the last 23 months.

I'd rather add Requires: perl-* at lcov.git.

Reply:
Yes, because Tizen does not have the latest `lcov` package, we need to use our repositories or cached RPM files for `lcov` and related ones.

FYI, I tested using my repositories:
- lcov: https://github.com/dongju-chae/lcov
- perl-PerlIO-gzip: https://github.sec.samsung.net/dongju-chae/perl-PerlIO-gzip
- perl-JSON: https://github.sec.samsung.net/dongju-chae/perl-JSON

I thought that this flag is enabled in our CI server only, it's okay to put there RPM files to cached one (e.g., `GBS-ROOT-SNAPSHOT/local/repos/tizen/x86_64/RPMS/`)